metastore/src/java/org/apache/hadoop/hive/metastore/ReplChangeManager.java

```diff
@@ -228,13 +228,10 @@ static public void setCmRoot(Path cmRoot) {
    *   to a deterministic location of cmroot. So user can retrieve the file back
    *   with the original location plus checksum.
    * @param conf
-   * @param checkSum checksum of the file, can be retrieved by {@link getCksumString}
-   * @return
-   * @throws IOException
-   * @throws MetaException
+   * @param checkSum checksum of the file, can be retrieved by {@link #checksumFor(Path, FileSystem)}
+   * @return Path
    */
-  static Path getCMPath(Configuration conf, String checkSum)
-      throws IOException, MetaException {
+  static Path getCMPath(Configuration conf, String checkSum) throws IOException, MetaException {
```

Comment:
Just noticed that those exceptions are not thrown and as we change this part of code, thought may be make those changes as well. If you don't want to change it now, that's fine.